keras_cv/layers/preprocessing/with_mixed_precision_test.py

```diff
@@ -156,14 +211,43 @@ def test_can_run_in_mixed_precision(self, layer_cls, init_args):
                     "operations `tf.image.adjust_saturation`, and "
                     "`tf.image.adjust_hue`. Skipping."
                 )
+            if layer_cls in NO_BOUNDING_BOXES_TESTS:
```

Comment:
Our GPU testing caught this little bug for us -- this only gets checked if there's no GPU present because this is over-indented by one level.

But this also makes the NO_BOUNDING_BOX tests no-ops anyway. How about instead, we check if layer_cls is in `NO_BOUNDING_BOX_TESTS` and if it is we exclude the bounding boxes from the inputs to the layer instead? (So basically just an `if` statement down around line 250

Reply:
Yes, that makes sense!

Reply:
CI is still unhappy 😢 I made the change at line 250 as you suggested.

Reply:
This line should not be here I think -- we can/should run these tests regardless of whether we have a GPU.

Reply:
Unresolving this -- we should get rid of this check on line 214, as these tests should still be run in this case.

Reply:
Oh okay

Reply:
Hello, @ianstenbit, I removed the check on line 214 as you suggested. Sorry for the infinite little errors towards the end😢